src/error.rs

```diff
@@ -0,0 +1,21 @@
+use anyhow::Error as AnyhowError;
+
+use fluvio::FluvioError;
+use serde_json::Error as SerdeJsonError;
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum NatsConnectorError {
```

Comment:
This type is not used anywhere in the code, right? We can remove it in favor of `anyhow` .

Reply:
Still not used

Reply:
removed